scripts/requirements.txt

```diff
@@ -6,3 +6,4 @@ influxdb
 pyhocon
 pyjq
 natsort
+git+https://github.com/rchain/pyrchain
```

Comment:
recommend you use `git+https://github.com/rchain/pyrchain@commit_hash`. Otherwise, this might not work if pyrchain master branch changes

Reply:
https://github.com/rchain/rchain-testnet-node/pull/17/commits/70ed739bac26bcb6ff383de9c062cb712b950ef8